openfisca_france/model/revenus/remplacement/indemnites_journalieres_securite_sociale.py

```diff
@@ -74,3 +73,9 @@ def function(self, simulation, period):
         )
 
         return period, result
+
+class date_arret_de_travail(Variable):
+    column = DateCol(default = nan)
```

Comment:
Oh, sorry, I didn't notice this. Which api calls crash ? In mes-aides, it seems fine.
The usual default date in openfisca is `1970-01-01`, which is not not always satisfying.

In that case I needed to be able to know if the date was defined, so a default date is not really good enough (unless we put `3000-01-01`, but that's dirty').

Maybe we can try to replace `nan` by `datetime64('NaT'))` ? @benjello do you know if there is an equivalent of + infinity for dates in numpy ?